src/psycop/common/model_evaluation/binary/time/timedelta_data.py

```diff
@@ -182,7 +182,7 @@ def get_sensitivity_by_timedelta_df(
     """
     df = get_timedelta_df(
         y=y,
-        y_hat=y_pred,
+        y_hat_probs=y_hat_probs,
```

Comment:
```suggestion
        y_hat_probs=y_hat_probs,
```
Maybe generalise the name? E.g. "prediction" or something, which is then just carried over?

Reply:
i don't understand 😢